charts/nextcloud/1.6.1/questions.yaml

```diff
@@ -237,17 +236,33 @@ questions:
                 label: "Postgres Data Volume Name"
                 schema:
                   type: string
+                  hidden: true
                   $ref:
                     - "normalize/ixVolume"
                   default: "ix-postgres_data"
+                  show_if: [["hostPathEnabled", "=", false]]
                   editable: false
+
               - variable: mountPath
                 label: "Postgresql Data Mount Path"
                 description: "Path where the volume will be mounted inside the pod"
                 schema:
                   type: path
-                  editable: false
```

Comment:
We should add `hidden` attr but not remove this one as we don't want the user to modify this